diesel_codegen/Cargo.toml

```diff
@@ -20,6 +20,7 @@ quasi = { verision = "^0.3.8", default-features = false }
 quasi_macros = { version = "^0.3.9", optional = true}
 syntex = { version = "^0.22.0", optional = true }
 syntex_syntax = { version = "^0.22.0", optional = true }
+diesel = { path = "../diesel" }
```

Comment:
Annoyingly, this means pointing at git for `diesel_codegen` won't work. However, the compile-fail tests will blow up if I point this at git, as the linker will find two crates to satisfy the `diesel` dependency (the compile fail tests just look in the build dir for dependencies, they don't have anything fancy like cargo). Basically our options are:

- Just accept this, and limit people to using the released version of codegen
- Use `sed` on Travis to rewrite this line (downside is that running tests locally will blow up by default)
- Rewrite `compiletest_rs` to use Cargo